packages/@react-spectrum/s2/intl/hr-HR.json

```
@@ -17,6 +18,8 @@
   "label.(optional)": "(opcionalno)",
   "label.(required)": "(obvezno)",
   "menu.moreActions": "Dodatne radnje",
+  "notificationbadge.indicatorOnly": "Nova aktivnost",
+  "notificationbadge.plus": "{notifications}+",
```

just noticed, should these all be this?
```suggestion
  "notificationbadge.plus": "{notifications, number}+",
```

---

ah, so the reason why i didn't do `number` was because in locales like `zh-CN-u-nu-hanidec`, that would return `NaN`

---

@yihuiliao @snowystinger I'll leave this as is, please let me know if further update is needed